README.md

```diff
@@ -91,6 +91,7 @@ get({
 
   // simple-get accepts all options that node.js `http` accepts
   // See: http://nodejs.org/api/http.html#http_http_request_options_callback
+  // Special option `noDecompress` allows to skip response body decompression.
```

Comment:
Can you add a new section to the readme, similar to the others that already exist there, to document how to use this? This one line is insufficent.